policyengine_us/parameters/gov/states/mi/tax/income/exemptions/filer.yaml

```diff
@@ -0,0 +1,21 @@
+description: Michigan provides this exemption amount for the filer head and spouse.
+values:
+  2017-01-01: 4_000
+  2018-01-01: 4_050
+  2019-01-01: 4_400
+  2020-01-01: 4_750
+  2021-01-01: 4_900
+  2022-01-01: 5_000
+  2023-01-01: 5_400
+
+metadata:
+  period: year
+  unit: currency-USD
+  label: Michigan income exemption amount
+  reference:
+  - title: Michigan Legal Code Section 206.30 - (2)
```

Comment:
This part of the legal code doesn't show up in the tax form. Do you know why?
>if a joint return is not made by the taxpayer and his or her spouse, the taxpayer may claim a personal exemption for the spouse if the spouse, for the calendar year in which the taxable year of the taxpayer begins, **does not have any gross income** and is not the dependent of another taxpayer.

Reply:
We have looked through the tax forms and there is nothing that accounts for this scenario. Do we want to model that using `spouse_separate_adjusted_gross_income` ? If so, how do we account for a spouse if the return is field seprately?

Reply:
@Mshuning we decided to not model that case, can ignore this